migrate-mongo-store.js

```diff
@@ -0,0 +1,138 @@
+/**
+ * Crowi::migrateStore
+ *
+ * @package Crowi
+ * @author  otofune <otofune@gmail.com>
+ */
+
+const mongoose = require('mongoose')
+
+const crowi = new (require('./lib/crowi'))(__dirname, process.env)
+
+const migrationSchema = new mongoose.Schema({
+  title: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  at: {
+    type: Date,
+    required: true,
+    index: true,
+  },
+  status: {
+    type: String,
+    required: true,
+    enum: ['up', 'down'],
+  },
+  history: [
+    {
+      status: {
+        type: String,
+        required: true,
+        enum: ['up', 'down'],
+      },
+      at: {
+        type: Date,
+        required: true,
+      },
+    },
+  ],
+})
+migrationSchema.methods.toEntry = function() {
+  const m = this
+  const entry = {
+    title: m.title,
+    timestamp: m.status === 'up' ? m.at.getTime() : null,
+  }
+  return entry
+}
+migrationSchema.statics.updateByEntry = function(entry) {
+  const { title } = entry
+  const at = entry.timestamp || Date.now()
+  const status = entry.timestamp === null ? 'down' : 'up'
```

Comment:
if timestamp is undefined or 0, status will be `up`. Is that your intention?

Reply:
I saw `tj/node-migrate`s implement: https://github.com/tj/node-migrate/blob/master/lib/migrate.js#L56-L57
I want to track 'down' event to know execution from another managers, but there are no hint without timestamp from store 😭

Reply:
I see, I'll use it! Thank you 🙏